recipes/jeepney/meta.yaml

```diff
@@ -0,0 +1,47 @@
+{% set name = "jeepney" %}
+{% set version = "0.3.1" %}
+{% set sha256 = "a6f2aa72e61660248d4d524dfccb6405f17c693b69af5d60dd7f2bab807d907e" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  skip: True  # [py<35]
+  script: flit install --deps none
+
+requirements:
+  build:
+    - python
+    - flit
```

Comment:
Technically any package that is built like this should also have `pip` as a build dependency, because `flit install` builds a wheel and then calls `pip` to install it.

Reply:
I know, but when you call `flit install`, that builds a temporary wheel and then invokes `pip` to install it. So `pip` is implicitly a build dependency. You get away without it because I think conda includes pip in environments by default.

Reply:
conda-forge actually doesn't automatically include `pip`, but our `flit` recipe does require it: https://github.com/conda-forge/flit-feedstock/blob/46ba733f246504bde590dba99808fac78820e014/recipe/meta.yaml#L35